CODEOWNERS

```diff
@@ -80,7 +80,7 @@
 /bundles/org.openhab.binding.lirc/ @kabili207
 /bundles/org.openhab.binding.logreader/ @paulianttila
 /bundles/org.openhab.binding.loxone/ @ppieczul
-/bundles/org.openhab.binding.lutron/ @actong
+/bundles/org.openhab.binding.lutron/ @actong @bobadair
```

Comment:
I think - compared to the original contribution - this PR adds only a small amount of code. Please leave the original line in place.

Reply:
Yes, I agree there isn't much in this commit. :-) It's the 12 that went before that I was really referring to.  I think I'm really the only one working on this binding in the past year or so.  I'll change it back if you think I should, though.  I don't want to upset anyone.